indico/modules/events/persons/controllers.py

```diff
@@ -257,47 +257,72 @@ def _process(self):
                                                has_predefined_affiliations=Affiliation.query.has_rows())
 
 
-class RHEmailEventPersons(RHManageEventBase):
+class RHEmailEventPersonsPreview(RHManageEventBase):
+    """Preview an email with EventPersons associated placeholders."""
+
+    @use_kwargs({
+        'body': fields.String(load_default=''),
+        'subject': fields.String(load_default=''),
+    })
+    def _process(self, body, subject):
+        person = self.event.persons[0]
+        email_body = replace_placeholders('event-persons-email', body, event=self.event, person=person)
+        email_subject = replace_placeholders('event-persons-email', subject, event=self.event, person=person)
+        tpl = get_template_module('events/persons/emails/custom_email.html', email_subject=email_subject,
+                                  email_body=email_body)
+        return {
+            'subject': tpl.get_subject(),
+            'body': tpl.get_body()
+        }
```

Comment:
```suggestion
        return jsonify(subject=tpl.get_subject(), body=tpl.get_body())
```

I don't think we rely on the (somewhat new) implicit JSON encoding behavior of Flask when returning a dict anywhere else